tmt/utils.py

```diff
@@ -45,6 +45,7 @@
 # Default name and order for step plugins
 DEFAULT_NAME = 'default'
 DEFAULT_PLUGIN_ORDER = 50
+DEFAULT_PLUGIN_ORDER_MULTIHOST = 65
```

Comment:
With regards to our recent discussion about the most suitable default orders, @lukaszachy, @thrix, does it make sense to you to place multihost prepare step before required packages installation? Would be good to choose the best possible default here.

Reply:
Let's go with `10` for now, this should be reasonably early.